src/compile-string.ts

```diff
@@ -26,6 +26,7 @@ export default function compileToString(str: string, config: EtaConfig): string
     (config.include ? ',include=E.include.bind(E)' : '') +
     (config.includeFile ? ',includeFile=E.includeFile.bind(E)' : '') +
     '\nfunction layout(p,d){__l=p;__lP=d}\n' +
+    (config.globalAwait ? 'let prs = [];\n' : '') + 
```

Comment:
Do you think we could rename this to `_prs`? That is something I have seen many tools which need to add additional variables use so it doesn't conflict with the actual code.